terraform/modules/public-load-balancer/main.tf

```diff
@@ -4,6 +4,13 @@ resource "aws_lb" "public" {
   load_balancer_type = "application"
   security_groups    = [aws_security_group.public_alb.id]
   subnets            = var.public_subnets
+  
+  tags = merge(
+    var.additional_tags,
+    {
+      name      = "${var.app_name}-lb"
```

Comment:
it doesn't have the workspace interpolated in comparison to the previous tags above and name of the resource too